cognite/client/data_classes/transformations/common.py

```diff
@@ -27,7 +27,8 @@ def dump(self, camel_case: bool = False) -> Dict[str, Any]:
 
         needs_dump = set(iterable_to_case(("view", "edge_type"), camel_case))
         for k in needs_dump.intersection(ret):
-            ret[k] = ret[k].dump(camel_case=camel_case)
+            if ret[k] is not None:
+                ret[k] = ret[k].dump(camel_case=camel_case)
```

Comment:
@vuhainguyen @silvavelosa This literally can't happen. `basic_obj_dump` will remove all keys where the corresponding value is `None`...

Reply:
Just kidding, there are two "basic obj dump" functions 🤣 My bad... 🤦

Reply:
I would recommend switching to from `_base.py`:

```py
def basic_instance_dump(obj: Any, camel_case: bool) -> Dict[str, Any]:
    # TODO: Consider using inheritance?
    dumped = {k: v for k, v in vars(obj).items() if v not in EXCLUDE_VALUE and not k.startswith("_")}
    if camel_case:
        return convert_all_keys_to_camel_case(dumped)
    return dumped
```

Reply:
Clean-up PR incoming 🤓